text/0028-dependency-server-licenses.md

```diff
@@ -0,0 +1,53 @@
+# Add License Field to the Dependency Server Metadata
+
+Supersedes [RFC #0009](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0009-dep-server.md)
+
+## Summary
+
+Add a field for license information into the dependency server schema.
+
+## Motivation
+
+Currently, the dependency server offers a lot of valuable information about the
+files it hosts. However, it currently does not have the capacity to supply
+license information. This field should be added to the dependency server
+because information about the licesnse(s) of a dependency is a valuable and
+necessary piece of information for those that are trying to make sure that
+thier apps use license that comply with thier use cases.
+
+## Implementation
+
+The following field should be added to the dependency server output:
+
+* `licenses`: an array of licenses that are used by the dependency
```

Comment:
Can you make sure to include a specification of what the acceptable values for this field are?